### ISSUE TITLE
Full CRUD would be nice

### ISSUE THREAD

**Issue by abonhote** [eclipse-store/bookstore-demo#4]
Currently, we can only insert new books - we can't edit them, nor delete them.

**Comment by changchengqin** [comment]
Yes. Looking forward to a complete CRUD example with multiple object relationships.

In actual projects, there are still many principles that need to be followed, such as:
1. Avoid establishing bidirectional relationships between models
2. The official manual document provides several best practices, one of which is:
[Use Immutable data models](https://docs.eclipsestore.io/manual/storage/storing-data/best-practice.html). 
Because it can improve performance. However, if you want to update these Immutable data models, other objects that reference them need to be manually synchronized and stored.

I hope the project team can share more practical experience and skills, including modeling, modification, synchronization, updating, etc.

thks.
